src/stackdriver-nozzle/nozzle/heartbeat.go

```diff
@@ -0,0 +1,38 @@
+package nozzle
+
+import (
+	"time"
+
+	"github.com/cloudfoundry/lager"
+)
+
+type Heartbeater interface {
+	Start()
+}
+
+type heartbeat struct {
+	logger lager.Logger
+	trigger chan time.Time
+	counter chan struct{}
+}
+
+func NewHeartbeat(logger lager.Logger, trigger chan time.Time, counter chan struct{}) Heartbeater {
+	return &heartbeat{logger: logger, trigger: trigger, counter: counter}
+}
+
+func (h *heartbeat) Start() {
+	go func() {
+		eventCount := 0
+		for {
```

Comment:
the thought was that this would run for the lifetime of the app. Who would be calling Stop on the Heartbeat, and when?